src/main/java/com/denizenscript/clientizen/debuggui/DebugOptionsMenu.java

```diff
@@ -81,6 +88,21 @@ public BooleanOptionButton(String id, BooleanSupplier getter, BooleanConsumer se
 		public void updateLabel() {
 			setLabel(prefix.copy().append(getter.getAsBoolean() ? ON : OFF));
 		}
+
+		@Override
+		public void addTooltip(TooltipBuilder builder) {
+			String translated = tooltip.getString();
+			if (!CoreUtilities.contains(translated, '\n')) {
+				builder.add(tooltip);
+				return;
+			}
+			List<String> splitLines = CoreUtilities.split(translated, '\n');
+			OrderedText[] lines = new OrderedText[splitLines.size()];
+			for (int i = 0; i < splitLines.size(); i++) {
+				lines[i] = Text.literal(splitLines.get(i)).asOrderedText();
+			}
+			builder.add(lines);
```

Comment:
Can't you just add to the builder for each splitLines individually?

Reply:
Good point, not 100% sure what I was thinking when I did that - I remember complaining it takes an array and not a varargs so might've been an IDE issue or something